types/react-text-mask/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for react-text-mask 16.0
+// Project: https://github.com/text-mask/text-mask
+// Definitions by: Guilherme Hübner <https://github.com/guilhermehubner>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import * as React from "react";
+
+type maskArray = Array<string | RegExp>;
+
+interface MaskedInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+    mask?: maskArray | ((input: HTMLInputElement) => maskArray);
+
+    guide?: boolean;
+
+    placeholderChar?: string;
+
+    keepCharPositions?: boolean;
+
+    pipe?: (conformedValue: string, config: any) => false | string | { value: string, indexesOfPipedChars: number[] };
+
+    onReject?: (infos: { conformedValue: string, maskRejection: boolean, pipeRejection: boolean }) => void;
+
+    onAccept?: () => void;
+}
+
+interface conformToMaskResult {
+    conformedValue: string;
+    meta: {
+        someCharsRejected: boolean
+    };
+}
+
+export class MaskedInput extends React.Component<MaskedInputProps, any> {}
```

Comment:
It doesn't look like this is available as a named export. This should probably be `export default class MaskedInput extends React.Component<MaskedInputProps, any> {}`.

Reply:
@andy-ms  You're right! https://github.com/text-mask/text-mask/blob/master/react/src/reactTextMask.js#L6 :sweat_smile: 

I've fix it :+1: